src/lib/features/listening-history/model/store.ts

```diff
@@ -12,16 +26,36 @@ export const listeningHistory = persistentWritable<number[]>([], {
  * Function that adds a podcast id to the beginning of listening history.
  * It removes any previous occurrences of the same podcast.
  *
- * @param podcastId {string} – an URl to add to history
+ * @param podcastId {string} – an id to add to history
  */
-export const addToListeningHistory = (podcastId: number) => {
+export const addToLocalListeningHistory = (podcastId: number) => {
   listeningHistory.update((history) => {
     return [podcastId, ...history.filter((id) => id !== podcastId)].slice(0, 6);
   });
 };
 
-currentlyPlayingEpisode.subscribe(($episode) => {
+/**
+ * Function that add a row into history SB table.
+ *
+ * @param podcastId {string} – an id to add to history
```

Comment:
nitpick (non-blocking): this type in the JSDoc is incorrect

suggestion: let's just remove this entire line, it doesn't add clarity to the comment